classWork/PhotoEnhancer/PhotoEnhancer/Filters/Transform/LeftParameters.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class LeftParameters:IParameters
```

Comment:
Неудачное название параметров (левые параметры) как следствие неудачного названия трансформера. Лучше так
```suggestion
    public class LeftSkewParameters:IParameters
```